api/src/api/userRouter.test.ts

```diff
@@ -50,6 +50,7 @@ describe("userRouter", () => {
   let stubUpdateLicenseStatus: jest.Mock;
   let stubUpdateRoadmapSidebarCards: jest.Mock;
   let stubUpdateOperatingPhase: jest.Mock;
+  let stubTaxEncryptionDecryptionClient: jest.Mocked<EncryptionDecryptionClient>;
```

Comment:
same comment going to be all over the place - no need for the word "Tax" in this variable name (and it is technically not accurate)